_includes/manuals/nightly/2.1_quickstart_installation.md

```diff
@@ -43,7 +44,7 @@ sudo yum-config-manager --enable rhel-7-server-optional-rpms rhel-server-rhscl-7
   <p>If you're using RH Satellite 5, you should instead sync and enable the channel there.</p>
 </div>
 
-<div class="quickstart_os quickstart_os_rhel7 quickstart_os_el7">
+<div class="quickstart_os quickstart_os_rhel7 quickstart_os_other_el7 quickstart_os_centos7">
```

Comment:
I think you should remove EPEL and foreman-release from this section, otherwise you end up with duplicated instructions I think.

Reply:
This section appears to just be Puppet setup.

Reply:
Oh right. I think Github diffs were hard to read and missed the RHEL7 header.